wormhole-connect/src/routes/cctpManual/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './cctpManual';
+export * from './utils';
```

Comment:
One thing I worry about with always using `export *` is that it won't be clear when identically named exports will get clobbered lower down (like in `routes/index.ts`).

Reply:
fair, not sure how I'd want to tackle that.  Gonna resolve for now and we can come back to it later